### DISCUSSION TITLE
Open apps and play music - Abrir apps y reproducir música

### DISCUSSION THREAD

**Discussion by rodalvis-sys** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#806] [General]
Is it possible to open my desktop apps like I would on my smartphone? If not, is it a function that could be seen in the near future or does it come out of your hands?

------------------------------------------------------------------------------------------------------------------------------------------------------

¿Es posible abrir mis aplicaciones de escritorio como lo haría en el teléfono inteligente? sino es así ¿Es una función que se podría ver en un futuro proximo o sale de sus manos?

**Comment by jrush-vj** [comment]
I am having the same issue
